tests/load/setup_k8s.sh

```diff
@@ -98,13 +99,16 @@ do
     case $response in
         create) #Setup Kubernetes Cluster
             echo -e "==================== Creating the GKE cluster "
-            # The total-max-nodes = WORKER_COUNT + 1 (MASTER)
-            $GCLOUD container clusters create $CLUSTER --region $REGION --scopes $SCOPE --enable-autoscaling --total-min-nodes "1" --total-max-nodes "301" --scopes=logging-write,storage-ro --addons HorizontalPodAutoscaling,HttpLoadBalancing  --machine-type $MACHINE_TYPE
+            $GCLOUD container clusters create $CLUSTER --region $REGION --scopes $SCOPE --enable-autoscaling --scopes=logging-write,storage-ro --addons HorizontalPodAutoscaling,HttpLoadBalancing --enable-dataplane-v2
```

Comment:
Are we missing `--machine-type=$MACHINE_TYPE` here cause I noticed an e2-medium is being assigned:
<img width="1619" alt="image" src="https://github.com/mozilla-services/autopush-rs/assets/3422688/d4e47adb-becc-40bf-a5d3-0f8e77563a09">

Reply:
Ah yeah, we could add something in if we wanted a machine type other than an e2-medium there. This node pool should only be used for the Kubernetes management stuff and the Locust master service deployment.

Reply:
I do not know the capacity of the e2-medium and if it can handle the locust-master. locust-master needs to handle a lot of networking with the workers, so tends to be hungry.